spec/csv_generator/csv_generator_spec.rb

```diff
@@ -29,7 +29,142 @@
       expect(headers).to eq(expected_headers)
     end
 
```

Comment:
Nothing to do with your changes but think there is an `expect(true).to be_truthy` above! Seems a bit redundant?